docs/development/translations.rst

```diff
@@ -0,0 +1,408 @@
+"SecureDrop"), and the translations for the desktop icons of the admin
+and journalist workstations used by news organizations (labeled "desktop").
+
+.. figure:: ../images/weblate/project.png
+   :align: center
+   :figwidth: 80%
+   :width: 100%
+
+   Components of the SecureDrop project in Weblate. Click on the image to see it full-size.
+
+Once a string is translated, it must be approved by a :ref:`reviewer <weblate_glossary_reviewer>`
+before being accepted into SecureDrop.
+
+Approved strings can only be modified by :ref:`reviewers <weblate_glossary_reviewer>`.
+However, translators can still suggest modifications and make comments
+if they notice something wrong.
+SecureDrop translations are a collaborative endeavour!
+
+Learn :ref:`how to translate SecureDrop using Weblate <how_to_translate_a_language_weblate>`.
+
+.. _getting_help_translations:
+
+Getting help
+^^^^^^^^^^^^
+
+Should you need help, you can do one of the following:
+
+* Post a message in the `translation category of the SecureDrop forum`_
+* Chat in the `SecureDrop instant messaging channel`_
+* Read the `Weblate documentation`_
+
+Background Information
+----------------------
+
+.. _what_is_securedrop:
+
+What is SecureDrop?
+^^^^^^^^^^^^^^^^^^^
+
+SecureDrop is a system that lets people share sensitive information
+with investigative journalists anonymously and securely. It's designed
+to protect its users with strong cryptography and network
+communications that hide locations and activity. For more information:
+
+* Learn about :doc:`what makes SecureDrop unique <../what_makes_securedrop_unique>`..
+* Watch `The Globe and Mail guide to using SecureDrop <https://www.youtube.com/watch?v=oSW2wMWtAMM>`_.
+* Read the `Localization Lab "Ask Me Anything" on SecureDrop <https://www.localizationlab.org/blog/2018/4/20/4bp1j2olispup45z8o2mm5nb5snxm2>`_.
+
+.. _who_uses_securedrop:
+
+Who uses SecureDrop?
+^^^^^^^^^^^^^^^^^^^^
+
+There are two kinds of SecureDrop users: :doc:`Sources <../source>`
+and :doc:`Journalists <../journalist>`. A source is an individual who
+wants to communicate securely and anonymously with a
+journalist. Sources are not expected to have any technical
+background. Journalists using SecureDrop have usually received proper
+training and understand the basic workflow of SecureDrop.
+
+.. _how_is_securedrop_translated:
+
+How is SecureDrop translated?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SecureDrop is translated using the **Weblate** platform. :ref:`Get started using Weblate <get_started_using_weblate>`
+
+Sources, journalists and admins use localized versions of SecureDrop. A malicious actor could attempt to modify their behavior by creating misleading translations. In order to mitigate that risk, all translations must be reviewed and accepted by designated :ref:`reviewers <weblate_glossary_reviewer>` before they become part of SecureDrop.
+
+.. _adding_a_new_language:
+
+Can new languages be added to SecureDrop?
```

Comment:
My intuition would be to also move this to the how-to section, because that's where I would look for it.

Reply:
I moved it.